app/services/flow/base_step.rb

```diff
@@ -28,6 +28,11 @@ def mark_step_incomplete(step = nil)
       nil
     end
 
+    def step_is_not_complete?(step = nil)
+      klass = step.nil? ? self.class : steps[step]
+      !flow_session[klass.to_s].present?
+    end
```

Comment:
Can you check the flow_session directly for the value you need where this is called rather than adding more metaprogramming for the flow_session?

Reply:
If you want to do this, you could replace `step_is_not_complete?(:address)` in `app/services/idv/steps/in_person/state_id_step.rb` with `flow_session[steps[:address].to_s].blank?`

Reply:
Thanks for the suggestions @tomas-nava! Tested and working as expected. Resolved with commit 1b8b956f098f2a227ca8b7cb62fd0d555c027ab3